Automattic/jetpack
# Blocks: remove external link style overrides

## Proposed changes:

This override was added in #19564 to address an issue that had not been fixed in core yet. It now has been, so it's safe to remove the work-around from the Jetpack plugin.

Internal reference:
- p1715759008450999-slack-C0D96691V

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Posts > Add New
* Insert an Eventbrite block
    * The External Link icon displayed after the support link should be displayed properly, aligned with the text.
    * This should work whether you use the Gutenberg plugin or not, and whether you use WP 6.4 or WP 6.5.
